rom/dev/src/flow/cold_reset/x509.rs

```diff
@@ -12,80 +12,19 @@ Abstract:
 
 --*/
 use crate::cprintln;
-use crate::crypto::{Crypto, PubKey};
+use crate::crypto::Crypto;
 use crate::rom_env::RomEnv;
+use caliptra_common::{
+    crypto::{self, PubKey},
+    x509,
+};
 use caliptra_drivers::*;
 use core::mem::size_of;
-use core::usize;
-use zerocopy::AsBytes;
 
 /// X509 API
 pub enum X509 {}
```

Comment:
Ditto here about eliminating this empty `enum`.

Reply:
Udpated.